Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py

```diff
@@ -155,11 +155,15 @@ def parse_incident_from_finding(message, parse_body_as_json=False):
 def fetch_incidents(aws_client, aws_queue_url, max_fetch, parse_body_as_json):
     try:
         client = aws_client.aws_session(service='sqs')
+        last_receipt_handles = demisto.getLastRun().get('lastReceiptHandles')
```

Comment:
Please add a short docstring explaining the way this works:
1. fetch.
2. create-incidents (while skipping previous fetch).
3. save fetch results to context.
4. try to delete all messages (if not successful, will continue next run).